frontend/src/components/common/header/components/menu-popup/menu-popup.tsx

```diff
@@ -6,7 +6,7 @@ import { authActions } from 'store/actions';
 import styles from './styles.module.scss';
 
 type Props = {
-  onClose: () => void;
+  onClose: (e: React.MouseEvent | void) => void;
```

Comment:
```suggestion
  onClose: (evt: React.MouseEvent) => void;
```

Reply:
void is needed for `dispatch(authActions.logout()).unwrap().then(onClose);` as it does not pass any event